doc/development.rst

```diff
@@ -86,6 +86,8 @@ This is the entire ABC bus class with all internal methods:
 .. autoclass:: can.BusABC
     :private-members:
     :special-members:
+    :noindex:
```

Comment:
This fixes some of the Sphinx errors, but adds
`WARNING: py:meth reference target not found: can.BusABC._apply_filters`
for all "private" methods.